__tests__/app.test.js

```diff
@@ -24,27 +24,12 @@ describe("GET /api/articles",() => {
                 expect(article.article_img_url).toEqual(expect.any(String));
             })
             expect(article.length).toBe(13)
-            expect(article.body).toBe(undefined)
-        })
-    })
-    test("status: 200, articles should be sorted in descending order", () => {
-        return request(app)
-        .get("/api/articles")
-        .expect(200)
-        .then(({ body: { article }}) => {
+            expect(article.hasOwnProperty('body')).toBe(false)
```

Comment:
Careful, this is outside of your for each so will always be false as article may be undefined.